readme.txt

```diff
@@ -3,7 +3,7 @@ Contributors: nathanrice, studiopress, wpmuguru
 Donate link: https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=5553118
 Tags: genesis, genesiswp, studiopress
 Requires at least: 3.9
-Tested up to: 4.9.8
+Tested up to: 5.2.2
 Stable tag: 1.0.0
```

Comment:
Hey @ethan-schaffer just like in the `genesis-tabs`, this line would need to be updated as well.

```suggestion
Stable tag: 1.0.1
```

We also would need to update the version [here](https://github.com/studiopress/genesis-beta-tester/blob/release/1.0.1/genesis-beta-tester.php#L9).